dev/tests/acceptance/tests/functional/Magento/FunctionalTest/Config/Section/ProductStockOptionsSection.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../vendor/magento/magento2-functional-testing-framework/src/Magento/FunctionalTestingFramework/Page/etc/SectionObject.xsd">
+    <section name="ProductStockOptionsSection">
```

Comment:
This section shouldn't live under `Magento_Config` module.
Based on code I see that this configuration were defined in `Magento_CatalogInventory` module.
So please move this section to appropriate module.
https://github.com/magento-engcom/msi/blob/2.3-develop/app/code/Magento/CatalogInventory/etc/adminhtml/system.xml

Reply:
Done.